jobs/pipeline-jobs.yml

```diff
@@ -123,6 +123,14 @@
       - 'pipeline-template':
           branches: staging master
 
+- project:
+    name: fl-wtr-list
```

Comment:
This needs to match the github repo name and the serverless PROJECT_NAME/identifier.